sw/device/silicon_creator/lib/cfi.h

```diff
@@ -0,0 +1,188 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_CFI_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_CFI_H_
+
+#include "sw/device/lib/base/hardened.h"
+
+/**
+ * @brief Control Flow Integrity (CFI).
+ *
+ * The CFI_COUNTER macros provide utilities to implement forward-edge and
```

Comment:
Since we're gonna have one CFI header, and we may implement other CFI styles in the future, could we give this variant of CFI a specific name? E.g. `CFI_FUNC_COUNTERS` or something.